src/screens/Payouts/PayoutsList.res

```diff
@@ -13,9 +13,16 @@ let make = () => {
   let (filters, setFilters) = React.useState(_ => None)
   let defaultValue: LoadedTable.pageDetails = {offset: 0, resultsPerPage: 10}
   let pageDetailDict = Recoil.useRecoilValueFromAtom(LoadedTable.table_pageDetails)
+  let setPageDetails = Recoil.useSetRecoilState(LoadedTable.table_pageDetails)
   let pageDetail = pageDetailDict->Dict.get("Payouts")->Option.getOr(defaultValue)
   let (offset, setOffset) = React.useState(_ => pageDetail.offset)
 
+  let clearPageDetails = () => {
+    let newDict = pageDetailDict->Dict.toArray->Dict.fromArray
+    newDict->Dict.set("Payouts", defaultValue)
```

Comment:
And why we are manipulating a state like this? As pageDetailDict is an atomvalue it should have been done like this please try this - 
```
    setPageDetails(prev => {
      let newDict = prev->DictionaryUtils.copyOfDict
      newDict->Dict.set("Payouts", defaultValue)
      newDict
    })
```